api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -20,5 +20,7 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
+    
```

Comment:
You should probably get rid of these blank line additions

Reply:
i have removed the blank line additions

Reply:
I actually meant to get rid of the additions i.e. revert the changes since you are technically making no changes to the file. But no problem, Sorry on my part for not being more clear

Reply:
@wluyima i have fixed it, hope its okay now...requesting for your review